src/Testcontainers/Configurations/Modules/MessageBrokers/LocalStackTestcontainerConfiguration.cs

```diff
@@ -0,0 +1,37 @@
+namespace DotNet.Testcontainers.Configurations
+{
+  using DotNet.Testcontainers.Builders;
+  using JetBrains.Annotations;
+
+  /// <inheritdoc cref="TestcontainerMessageBrokerConfiguration" />
+  [PublicAPI]
+  public class LocalStackTestcontainerConfiguration : TestcontainerMessageBrokerConfiguration
```

Comment:
@bgener, is there any special reason to inherit the classes from *MessageBroker* classes?

Reply:
I do not think it is necessary, modules are moved to their own project and dependency in the next time. For now you can leave it there if you like.